fms_core/viewsets.py

```diff
@@ -235,10 +238,15 @@ def _list_keys(d: Dict[str, Any]) -> Dict[str, Any]:
     "email": FREE_TEXT_FILTERS,
 }
 
+_sample_kind_filterset_fields: FiltersetFields = {
+    "id": PK_FILTERS,
+    "name": CATEGORICAL_FILTERS_LOOSE,
+}
+
 _sample_filterset_fields: FiltersetFields = {
     "id": PK_FILTERS,
     "name": CATEGORICAL_FILTERS_LOOSE,
-    "biospecimen_type": CATEGORICAL_FILTERS,
+    "sample_kind": CATEGORICAL_FILTERS,
```

Comment:
Might be best to assign a FK_FILTERS to recognize it is not a local field... Not that it changes anything ...